doc/contribute/guidelines.rst

```diff
@@ -554,6 +563,31 @@ exceptions:
 * Avoid using binary literals (constants starting with ``0b``).
 * Avoid using non-ASCII symbols in code, unless it significantly improves
   clarity, avoid emojis in any case.
+* Use proper capitalization of nouns in code comments (e.g. ``UART`` and not
+  ``uart`` or ``CMake`` ``cmake``).
+
+Beyond C code, the following coding style rules apply to other types of files:
+
+* CMake
+
+  * Indent with spaces, indentation is two spaces.
+  * No space between commands (e.g. ``if``) and the corresponding opening
+    bracket (e.g. ``(``).
+
+* Devicetree
+
+  * Indent with tabs.
+  * No additional empty lines after ``};``.
```

Comment:
Does this mean we cannot have empty lines between node declarations? E.g. like this:
```

node1: node@1 {
     foo;
};
/* No empty line allowed here? */
node2: node@2 {
     bar;
};
```

If so, I don't think that's a good idea.

Reply:
@nordicjm @gmarull @bjarki-andreasen

Reply:
Hmm it's the wrong wording, it is meant to prevent this:
```
node1: node@1 {
    node2: node@2 {
         bar;
    }; <-- newline gap here is fine

    node2: node@2 {
         bar;
    };
<-- newline gap should not be here
};
```

Reply:
I agree with @henrikbrixandersen, it looks ugly.

Reply:
Maybe change to:
```
* No additional empty lines between a dedented `};``.
```
since
```
/ {
        node {

        };

        node {

        }; <- same level
}; <- dedents
```
dedents